templates/opam.mustache

```diff
@@ -1,15 +1,21 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer: "{{& opam-file-maintainer }}{{^ opam-file-maintainer }}palmskog@gmail.com{{/ opam-file-maintainer }}"
 
 homepage: "https://github.com/{{ organization }}/{{ shortname }}"
-dev-repo: "https://github.com/{{ organization }}/{{ shortname }}.git"
+dev-repo: "git+https://github.com/{{ organization }}/{{ shortname }}.git"
 bug-reports: "https://github.com/{{ organization }}/{{ shortname }}/issues"
 {{# license }}license: "{{ identifier }}"{{/ license }}
 
+synopsis: "{{& synopsis }}"
+description: """
+{{& description }}"""
+
 build: [make "-j%{jobs}%"]
 install: [make "install"]
 remove: ["rm" "-R" "%{lib}%/coq/user-contrib/{{ namespace }}"]
+flags: light-uninstall
 depends: [
+  "ocaml"
```

Comment:
Is this explicit dependency declaration actually useful?

Reply:
The addition of `"ocaml"` as a dependency was made by @gares and others when converting to OPAM 2.0 format. I assumed it was necessary. See an [example](https://github.com/coq/opam-coq-archive/commit/06dec98bcd02339d3f3990f8e9fe4dcdf87f0808).